File: src/main/java/com/softwareloop/glo/model/Rom.java
package com.softwareloop.glo.model;

import lombok.Data;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;

@Data
@XmlAccessorType(XmlAccessType.FIELD)
public class Rom {

    @XmlAttribute
    private String name;

    @XmlAttribute
    private Integer size;

    @XmlAttribute
    private String crc;

    @XmlAttribute
    private String md5;

    @XmlAttribute
    private String sha1;

}

File: src/main/java/com/softwareloop/glo/model/Rom.kt
package com.softwareloop.glo.model

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute

@XmlAccessorType(XmlAccessType.FIELD)
class Rom {

    @XmlAttribute
    val name: String? = null

    @XmlAttribute
    val size: Int? = null

    @XmlAttribute
    val crc: String? = null

    @XmlAttribute
    val md5: String? = null

    @XmlAttribute
    val sha1: String? = null

}